setup.cfg

```diff
@@ -1,8 +1,39 @@
-[bdist_wheel]
-universal = 1
```

Comment:
Since we've dropped support for Python 2, this isn't actually a universal wheel anymore. (azavea/django-amazon-ses#53)